src/main/kotlin/no/nav/dagpenger/inntekt/Configuration.kt

```diff
@@ -38,7 +46,11 @@ private val prodProperties = ConfigurationMap(
                 "database.name" to "dp-inntekt-db",
                 "vault.mountpath" to "postgresql/prod-fss/",
                 "application.profile" to "PROD",
-                "application.httpPort" to "8099"
+                "application.httpPort" to "8099",
+                "hentinntektliste.url" to "https://localhost/inntektskomponenten-ws/rs/api/v1/hentinntektliste",
```

Comment:
You might consider removing these "default" config values  from PROD. The application will then fail fast with a nice error message if these config values are missing from the environment.

Reply:
fixed!